### PR TITLE
Tkinter hello world tutorial and end-to-end demo. 

### PR DESCRIPTION
**Pull request by jlopez8**
We push an experimental **end-to-end handwriting-to-LaTeX demo** that uses tkinter's canvas to save a handwritten math image and sends it to the MathPix API for a LaTeX prediction.

### CODE REVIEW

**Comment by rkingery** [comment]
> In this PR, we are pushing two experimental directories, one for an end-to-end handwriting-to-LaTeX demo and one for a tkinter hello world app. The first is a nb showing how one can use tkinter's canvas to save a handwritten math image and send it to the MathPix OCR for a LaTeX prediction. The second is a tkinter tutorial useful for understanding how various widgets work I tkinter. The latter I advanced to include callbacks so that a canvas can be inserted using a dropdown menu and the image can also be saved through the same menu. Another callback allows you to enter an image save filename which works with the same file save callback.

Clean this up and make it more readable. When you have something like "first...second...third" it's better to use bullets or numbers. Remember these are markdown cells. Use them to your advantage. Communicate what this PR contains in as few words as possible, in as easy to read a way as possible. People hate reading dense walls of text.

**Comment by rkingery** [comment]
You're gonna hate this, but go back through *all* of your uploaded notebooks and make them PEP-8 compliant. You need to get familiar with this so much you don't even have to think about it, because it's strictly enforced in all industry python code.

https://www.python.org/dev/peps/pep-0008/

File: experiments/tkinter_hello_world_tutorial/temp.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import numpy as np
```
Comment (rkingery) [inline experiments/tkinter_hello_world_tutorial/temp.py:1]: What is this file for? If you don't need this, delete it or add to the gitignore.

**Review by rkingery — CHANGES_REQUESTED** [review]
See all comments.

File: experiments/tkinter_hello_world_tutorial/temp.py (hunk 1 above)
Comment (jlopez8) [reply experiments/tkinter_hello_world_tutorial/temp.py:1]: Removed.

**Review by rkingery — APPROVED** [review]
Looks good to me. Free to merge.

**Pull request merged**